AK/Forward.h

```diff
@@ -125,6 +125,27 @@ class WeakPtr;
 template<typename T, size_t inline_capacity = 0>
 requires(!IsRvalueReference<T>) class Vector;
 
+namespace Detail {
+// This cannot be in StdLibExtraDetail, due to the dependency on size_t
```

Comment:
You could use your `Decay` helper to do array-to-pointer conversion here and avoid needing `size_t`

```c++
template<typename T>
inline constexpr bool IsAnyString = IsSame<T, String> || IsSame<T, StringView> || IsSame<Decay<T>, char*> || IsSame<Decay<T>, char const*>;
```

Reply:
Decay needs size_t as well....

Reply:
I could move it to StringUtils though